pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -1072,6 +1072,166 @@ def plot_channel_parameter(self, param_name: str, **plt_kwargs: Any) -> plt.Figu
         )
         return fig
 
+    def get_ts_contribution_posterior(
+        self, var_contribution: str, original_scale: bool = False
+    ) -> DataArray:
+        """Get the posterior distribution of the time series contributions of a given variable.
+
+        Parameters
+        ----------
+        var_contribution : str
+            The variable for which to get the contributions.
```

Comment:
Does this work for all variables? Or is there only a few that work? Maybe list out

Reply:
Added better description in the doctring 👍